winui/Cartesian-Charts/Axis/Gridlines.md

```diff
@@ -97,21 +99,21 @@ chart.SecondaryAxis = new NumericalAxis();
 
 ## Minor Grid Lines
 
-Minor gridlines will be added automatically when the small tick lines is defined using the [SmallTicksPerInterval](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.RangeAxisBase.html#Syncfusion_UI_Xaml_Charts_RangeAxisBase_SmallTicksPerInterval) property of chart axis.
+Minor gridlines will be added automatically when the minor tick lines is defined using the [MinorTicksPerInterval]() property of chart axis.
```

Comment:
Minor gridlines will be added automatically when the minor tick lines is defined using the [MinorTicksPerInterval]() property of the chart axis.